spec/index.bs

```diff
@@ -6767,19 +6792,52 @@ interface GPUCanvasContext {
         given device.
 
         <div algorithm="GPUCanvasContext.getSwapChainPreferredFormat">
-            **Called on:** {{GPUCanvasContext}} this.
+            **Called on:** {{GPUCanvasContext}} |this|.
 
             **Arguments:**
-            <pre class=argumentdef for="GPUCanvasContext/getSwapChainPreferredFormat(device)">
-                device: Device the swap chain format should be queried for.
+            <pre class=argumentdef for="GPUCanvasContext/getSwapChainPreferredFormat(adapter)">
+                |adapter|: Adapter the swap chain format should be queried for.
             </pre>
 
-            **Returns:** {{Promise}}&lt;{{GPUTextureFormat}}&gt;
+            **Returns:** {{GPUTextureFormat}}
 
-            Issue: Describe {{GPUCanvasContext/getSwapChainPreferredFormat()}} algorithm steps.
+            <div class=content-timeline>
+                1. Let |preferredFormat| be the [$preferred swap chain format$] for |this| and |adapter|.
+                1. Return |preferredFormat|.
+            </div>
         </div>
 </dl>
 
+The <dfn dfn>guaranteed swap chain formats</dfn> are a [=set=] of {{GPUTextureFormat}}s that must be
+supported when specified as a {{GPUSwapChainDescriptor}}.{{GPUSwapChainDescriptor/format}} regardless
+of the given {{GPUSwapChainDescriptor}}.{{GPUSwapChainDescriptor/device}}, initially set to:
+«{{GPUTextureFormat/"bgra8unorm"}}, {{GPUTextureFormat/"bgra8unorm-srgb"}}, {{GPUTextureFormat/"rgba16float"}}».
```

Comment:
woah. Didn't realize the [infra spec](https://infra.spec.whatwg.org/#list) uses these characters. They actually put the unicode characters in their bikeshed doc. But perhaps we should use `&laquo; &raquo;` so people don't end up copying and pasting these around in the future?